src/mock_alchemy/comparison.py

```diff
@@ -28,6 +31,9 @@
     ALCHEMY_FUNC_TYPE,
     ALCHEMY_LABEL_TYPE,
 )
+if version.parse(sqlalchemy.__version__) >= version.parse("1.4.47"):
```

Comment:
Can we make this more extensible so that when additional statements are created per version, we can simply add them?